src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseAgreedSettledPartAdmitDefendantLipNotificationHandler.java

```diff
@@ -49,33 +55,64 @@ public List<CaseEvent> handledEvents() {
-            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
-        );
+        if (caseData.isRespondent1NotRepresented()) {
+            return Map.of(
+                RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1()),
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
+            );
+        } else {
+            return Map.of(
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                CLAIM_LEGAL_ORG_NAME_SPEC, getRespondentLegalOrganizationName(caseData)
+            );
+        }
+    }
+
+    private String getRespondentLegalOrganizationName(CaseData caseData) {
+        Optional<Organisation> organisation = organisationService.findOrganisationById(caseData.getRespondent1OrganisationId());
+        String respondentLegalOrganizationName = null;
+        if (organisation.isPresent()) {
+            respondentLegalOrganizationName = organisation.get().getName();
+        }
+        return respondentLegalOrganizationName;
     }
 
     private CallbackResponse notifyDefendantForPartAdmitClaimSettled(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        if (!caseData.isLRvLipOneVOne() || caseData.getRespondent1().getPartyEmail() == null) {
-            return AboutToStartOrSubmitCallbackResponse.builder()
-                .build();
+        String defendantEmailId = addEmail(caseData);
+        if (Objects.nonNull(defendantEmailId)) {
+            notificationService.sendMail(
+                defendantEmailId,
+                setUpEmailTemplate(caseData),
+                addProperties(caseData),
+                String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+            );
         }
-
-        notificationService.sendMail(
-            caseData.getRespondent1().getPartyEmail(),
-            setUpEmailTemplate(caseData),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
-
         return AboutToStartOrSubmitCallbackResponse.builder()
             .build();
     }
 
     private String setUpEmailTemplate(CaseData caseData) {
-        return caseData.isRespondentResponseBilingual()
-            ? notificationsProperties.getRespondentLipPartAdmitSettleClaimBilingualTemplate()
-            : notificationsProperties.getRespondentLipPartAdmitSettleClaimTemplate();
+        if (caseData.isRespondent1NotRepresented()) {
+            return caseData.isRespondentResponseBilingual()
+                ? notificationsProperties.getRespondentLipPartAdmitSettleClaimBilingualTemplate()
+                : notificationsProperties.getRespondentLipPartAdmitSettleClaimTemplate();
+        } else {
+            return notificationsProperties.getRespondentLrPartAdmitSettleClaimTemplate();
+        }
+    }
+
+    private String addEmail(CaseData caseData) {
```

Comment:
perhaps not addEmail but getEmail?

Reply:
also it looks like you have the same code else where. so perhaps it is better to have a method in caseData so that you could resuse it in several handlers instead of duplicating code

Reply:
Updated code for this